### DISCUSSION TITLE
Using an iPhone as a bonded connection

### DISCUSSION THREAD

**Discussion by bobradamus** [Ysurac/openmptcprouter#3685] [Q&A]
Hey!

I’ve tried everything on this. Please does anyone have it working where they’ve got an iPhone (15 max for me) included in their bonding setup?

How do you do it? I’ve tried USB, single usb to ethernet, double usb to ethernet (one in phone on in switch).

Ae there only certain IP addresses subnets you have to use for an iPhone?

If anyone has the time to write a short guide on how they made it work I’d really appreciate the help!

Thanks, Rob

**Comment by bobradamus** [comment]
c’mon - somebody must’ve got this working already - surely?

**Reply by Ysurac** [reply]
I don't have an iPhone, I can't really help on this.

**Comment by q2go** [comment]
Is USB tethering supported at all with OpenMPTCProuter? (Just asking, I am new to OMR myself.)

**Reply by Ysurac** [reply]
Yes it's supported.
You can find some doc from OpenWRT here: https://openwrt.org/docs/guide-user/network/wan/smartphone.usb.tethering (everything is already installed)

**Comment by q2go** [comment]
So you might be lucky and find help on https://forum.openwrt.org @bobradamus – at least it should be a good idea to give it a try.
